src/pages/LocationsPage.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import Location from '../components/Location'
+import data from '../data.json'
+
+/*LocationList.propTypes = {
+  locations: PropTypes.arrayOf(
+    PropTypes.shape({
+      image: PropTypes.string,
+      description: PropTypes.string,
+    })
+  ),
+}*/
+
+export default function LocationList() {
+  return (
+    <LocationWrapper>
+      {data.map(({ id, name, description, image }) => (
+        <Location
+          key={id}
+          name={name}
+          description={description}
```

Comment:
You pass the description here but you never receive it in your `Location` component!